cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -187,6 +193,26 @@ createLoggingLayer ver nodeConfig' p = do
                                                   (nodeBasicInfo nodeConfig p nodeStartTime)
            >>= loadPlugin switchBoard
 
+         -- Forward all the metrics/peers/errors to 'TraceForwarderBK' using 'mapBackends'.
+         let metricsLogger = "cardano.node.metrics"
+             peersLogger   = "cardano.node.metrics.peersFromNodeKernel"
```

Comment:
@denisshevchenko, `cardano.node.metrics.peersFromNodeKernel` should be implied by `cardano.node.metrics` -- is it really the case that you still need the former?

Reply:
Thanks, fixed.